vectortile/src/test/scala/geotrellis/vectortile/ProjectionSpec.scala

```diff
@@ -0,0 +1,41 @@
+package geotrellis.vectortile
+
+import geotrellis.proj4._
+import geotrellis.vector._
+import geotrellis.vectortile.internal._
+
+import org.scalatest._
+
+
+class ProjectionSpec extends FunSpec {
+  describe("VectorTile Projection Conversions") {
+    it("should read a point from a VectorTile and write it back") {
+      val p = Point(-61.3422, 10.4068)
```

Comment:
This coordinate was extracted from a live MVT using [vt2geojson[(https://github.com/mapbox/vt2geojson), so it uses this projection math:

https://github.com/mapbox/vector-tile-js/blob/58df1e9344ee64f26deee84a9f54cee11fb95ef6/lib/vectortilefeature.js#L137-L145

As a result, I'm not sure what value is correct for translation.

Reply:
I see. That's good to know. It was suggested that I do a third writing/read of the point to see if  it continues to shift. This is the code I used to run this test:

```scala
    it("should read a point from a VectorTile and write it back") {
      val p1 = Point(-61.3422, 10.4068)
      val reprojected = point.reproject(LatLng, WebMercator)
      val tileExtent =
        Extent(-6887893.4928338025, 1095801.2374962866, -6809621.975869782, 1174072.7544603087)
        //SpatialKey(168, 241).extent(ZoomedLayoutScheme(WebMercator).levelForZoom(9).layout)

      val f = MVTFeature(reprojected, Map.empty[String, Value])

      val layer = StrictLayer(
        name = "test",
        tileWidth = 128,
        version = 2,
        tileExtent = tileExtent,
        points = Seq(f),
        multiPoints = Seq.empty,
        lines = Seq.empty,
        multiLines = Seq.empty,
        polygons = Seq.empty,
        multiPolygons = Seq.empty
      )

      val vt = VectorTile(Map("test" -> layer), tileExtent)

      val vt2 = VectorTile.fromBytes(vt.toBytes, tileExtent)
      val p2 = vt2.layers("test").features.head.geom.reproject(WebMercator, LatLng)

      val vt3 = VectorTile.fromBytes(vt2.toBytes, tileExtent)
      val p3 = vt3.layers("test").features.head.geom.reproject(WebMercator, LatLng)

      println(s"\n\nThis was the initial point: $p1")
      println(s"This is the point after its been written: $p2")
      println(s"This is the point after its been written, read in, and written out again: $p3")
    }
```

And these are the results:

```
This was the initial point: POINT (-61.3422 10.4068)
This is the point after its been written: POINT (-61.33666992187498 10.401377554543565)
This is the point after its been written, read in, and written out again: POINT (-61.33117675781249 10.395974612177667)
```

So it appears that the geometry continues to be shifted to the up and left. This was using the transformation of a full point instead of a half point like you commented above. I wouldn't expect `p1` to be equal to `p2` or `p3`, but I'd expected `p2` and `p3` to be the same.

Reply:
With further exploration here, `Point(-61.347656249999986, 10.412183158667512)` is a good starting point (ha!) that round trips with half a point as the offset.